services/download/src/main/java/app/coronawarn/server/services/download/FederationGatewayDownloadService.java

```diff
@@ -82,6 +82,31 @@ public BatchDownloadResponse downloadBatch(String batchTag, LocalDate date) thro
     }
   }
 
+  /**
+   * Audit the batch from the EFGS for the given date.
+   *
+   * @param batchTag The batchTag of the batch that should be audited.
+   * @param date     The date for which the batch should be audited.
+   */
+  public void auditBatch(String batchTag, LocalDate date) {
+    try {
+      ResponseEntity<String> auditInformation = federationGatewayClient
+          .getAuditInformation(date.format(ISO_LOCAL_DATE), batchTag);
+      logger.info("Retrieved response from EFGS:{}", auditInformation);
```

Comment:
Do we need to log the whole response from EFGS on info level? It might be worth to log it only if the spring profile`debug` is enabled.